pkg/gate/gate.go

```diff
@@ -252,7 +252,7 @@ func Start(ctx context.Context, opts ...StartOption) error {
 	}
 
 	// Apply default telemetry config first
-	c.conf.Editions.Java.Config = *telemetry.WithDefaults(&c.conf.Editions.Java.Config)
+	c.conf = telemetry.WithDefaults(c.conf)
```

Comment:
These defaults will potentially overwrite any telemetry config  passed to the Start options.
Please check how we already define defaults [here](https://github.com/minekube/gate/blob/c5f087ec690e035c1bab74514aa1f6583f52d7c4/pkg/gate/config/config.go#L14)